doc/userguide/security.rst

```diff
@@ -0,0 +1,137 @@
+Security Considerations
+=======================
+
+Running as a User Other Than Root
+---------------------------------
+
+.. note:: If using the Suricata RPMs, either from the OISF COPR repo,
+          or the EPEL repo the following is already configured for
+          you. The only thing you might want to do is add your
+          management user to the ``suricata`` group.
+
+Many Suricata examples and guides will show Suricata running as the
+*root* user, particularly when running on live traffic. As Suricata
+generally needs low level read (and in IPS write) access to network
+traffic, it is required that Suricata starts as root, however Suricata
+does have the ability to drop down to a non-root user after startup
+which could limit the impact of a security vulnerability in Suricata
+itself.
+
+.. note:: Currently the ability to drop root privileges after startup
+          is only available on Linux systems.
```

Comment:
not even on freebsd?

Reply:
Never tried, I don't see libcap-ng in the ports, and our privilege dropping is dependent on that.  We have an open ticket with some work for generic support, but it needs to be revisited: https://redmine.openinfosecfoundation.org/issues/2931